### PR TITLE
StaticBatchInstallEventListener被实例化多次时防止重复部署

### PR DESCRIPTION
**Pull request by loong-coder**
## Summary by CodeRabbit

- **Refactor**
  - Corrected a field name from `isBatchdDeployed` to `isBatchedDeployed` for clarity and consistency.

- **Tests**
  - Enhanced testing for batch deployment with additional instance validation to ensure the integrity of the batch installation process.

### CODE REVIEW

File: koupleless-base-plugin/src/main/java/com/alipay/sofa/koupleless/plugin/manager/listener/StaticBatchInstallEventListener.java [hunk 1]
```diff
@@ -47,15 +42,15 @@
 public class StaticBatchInstallEventListener implements ApplicationListener<ApplicationReadyEvent> {
 
     // 合并部署是否已经完成，防止重复执行。
-    private AtomicBoolean isBatchdDeployed = new AtomicBoolean(false);
+    private static final AtomicBoolean isBatchedDeployed = new AtomicBoolean(false);
```
Comment (lvjing2) [inline koupleless-base-plugin/src/main/java/com/alipay/sofa/koupleless/plugin/manager/listener/StaticBatchInstallEventListener.java:45]: 这个变量原来写错了，正确的应该是 isBatchDeployed

**Comment by lvjing2** [comment]
还有格式化要做下，只要本地执行下 mvn clean package -DskipTests 命令就会自动格式化的
image

File: koupleless-base-plugin/src/main/java/com/alipay/sofa/koupleless/plugin/manager/listener/StaticBatchInstallEventListener.java (hunk 1 above)
Comment (loong-coder) [reply koupleless-base-plugin/src/main/java/com/alipay/sofa/koupleless/plugin/manager/listener/StaticBatchInstallEventListener.java]: 好的

**Review by lvjing2 — APPROVED** [review]
LGTM

**Pull request merged**